x-pack/plugins/security_solution/public/resolver/view/use_state_syncing_actions.ts

```diff
@@ -22,10 +23,11 @@ export function useStateSyncingActions({
   resolverComponentInstanceID: string;
 }) {
   const dispatch = useResolverDispatch();
+  const locationSearch = useLocation().search;
   useLayoutEffect(() => {
     dispatch({
       type: 'appReceivedNewExternalProperties',
-      payload: { databaseDocumentID, resolverComponentInstanceID },
+      payload: { databaseDocumentID, resolverComponentInstanceID, locationSearch },
```

Comment:
dispatch this action any time the `location.search` changes.

Reply:
Having this action fire whenever location.search changes seems sort of overkill, why not just when the relevant params change?

Reply:
The app will need to read the entire query string. For example, we might need to show a link that causes our panel to open. In that case we'll want the link to include all query string values (not just ones relevant to this Resolver instance.)